recipes/kaleido-core/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "kaleido-core" %}
+{% set version = "0.1.0a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_minimal_linux_x64.zip  # [linux]
+  sha256: 523fce13fca1fac2770273f2f19b3bac20d7a47e78ef9caa58590e98500535d8  # [linux]
+
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_mac.zip  # [osx]
```

Comment:
What's the minimum osx version supported?

Reply:
10.10

Reply:
Is there a dependency that can be used to specify this?

Reply:
Thanks!